### PR TITLE
Query endpoint

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: bedrock/test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2015 Digital Bazaar, Inc. All rights reserved.
+ */
+var bedrock = require('bedrock');
+// require('bedrock-permission');
```
Comment (dlongley) [inline bedrock/test.js]: Remove commented line now?

File: lib/services.credentials.js [hunk 2]
```diff
@@ -22,6 +23,10 @@ var BedrockError = bedrock.util.BedrockError;
 
 var logger = bedrock.loggers.get('app');
 
+// FIXME: update to use new permissions model
```
Comment (dlongley) [inline lib/services.credentials.js]: What's this FIXME about? Is that update done yet?

**Comment by dlongley** [comment]
If @davidlehn gives his consent on this one, merge it. Assigning to him.

**Comment by davidlehn** [comment]
It would be good to fill out the validator stub with actual validation.  It's a bit difficult to tell what bad filter input will do.  Otherwise this looks ok for now.

**Pull request merged**